task4/script.js

```diff
@@ -1,6 +1,8 @@
+const df = document.createDocumentFragment();
```

Comment:
Introduced "documentFragment".
I didn't know this.
It turns out that the number of DOM accesses is reduced and the speed is increased.